usaspending_api/api_contracts/contracts/award/idv/Download.md

```diff
@@ -0,0 +1,34 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# IDV Download [/api/v2/download/idvs/]
+
+This end point returns a zipped file containing IDV data.
+
+## POST
+
++ Request (application/json)
+    + Parameters
+        + `award_id`: `CONT_AW_9700_-NONE-_N0018918D0057_-NONE-` (required, string)
++ Response 200 (application/json)
+    + Attributes
+        + results (IDVDownloadResponse)
+
+# Data Structures
+
+## IDVDownloadResponse (object)
++ total_size: 35.055 (required, number)
```

Comment:
`total_size`, `total_rows`, `total_columns` need to be nullable

Reply:
:ok_hand: